src/SerializerFactory.php

```diff
@@ -67,6 +69,13 @@ private function shouldUseObjectsForMaps() {
 	}
 
 	/**
+	 * @return bool
+	 */
+	private function shouldSerializeSnaksWithHash() {
+		return !(bool)( $this->options & self::OPTION_OBJECTS_FOR_MAPS );
```

Comment:
![I'm not blocking the view - I am the view.](https://41.media.tumblr.com/tumblr_mcvdjxiPEa1rxgjtso1_500.jpg)

Jeroen doesn't see the typo for all the kittens on his screen.